### ISSUE TITLE
Question about the distribution of the new data

### ISSUE THREAD

**Issue by AngledLuffa** [ShuhengL/acl2023_conllpp#1]
Hi,

I was looking for a recent English newswire NER dataset, and I came across your paper and dataset.  My understanding from reading the text is that the annotated data is the test set, [conllpp.txt](https://github.com/ShuhengL/acl2023_conllpp/blob/main/dataset/conllpp.txt), but not training data.  Furthermore, the `conll_prime` data split does not have any of the newly annotated data mixed in to the training data.  Is that correct?

Thanks!

**Comment by ShuhengL** [comment]
Yes, that is correct!

`conllpp.txt` is the newly annotated test set that we provide. `conll_prime/` does not include anything from `conllpp.txt`, but rather the original CoNLL-2003 English NER dataset scrambled and regrouped.

Hope that helps :)

**Comment by AngledLuffa** [comment]
Excellent, thanks for the clarification.

Would you happen to know of any training set sized datasets which do recent English newswire?

**Comment by ShuhengL** [comment]
Unfortunately I do not think I am aware of any. It would be an interesting comparison to see how models trained on recent data perform on different test sets...

**Comment by AngledLuffa** [comment]
I will be sure to let you know if we find any!  Thank you for the interesting work.

BTW, would you want it posted at https://github.com/juand-r/entity-recognition-datasets ?   I think most of the datasets there are full sized datasets, but I also think a modern test set sized dataset would be a useful resource to add.
